pkg/backend/openshiftcluster.go

```diff
@@ -55,7 +55,8 @@ func (ocb *openShiftClusterBackend) try(ctx context.Context) (bool, error) {
 			ocb.m.EmitGauge("backend.openshiftcluster.workers.count", int64(atomic.LoadInt32(&ocb.workers)), nil)
 			ocb.cond.Signal()
 
-			ocb.m.EmitGauge("backend.openshiftcluster.duration", time.Now().Sub(t).Milliseconds(), map[string]string{
+			duration := time.Now().Sub(doc.CorrelationData.RequestTime).Seconds()
```

Comment:
this code needs to detect if a change in provisioningstate has taken place, and only if it has log the duration time.